features/people_page.feature

```diff
@@ -0,0 +1,15 @@
+Feature: Checking user name in people room
+
+Background:
+    Given we are on Hipchat Login Page
+    When we enter login
+    And we enter password
+    Then we see Welcome title
+    Given we are on Account settings Page
+    When we get full user name from settings
```

Comment:
Then?

Reply:
Agree with you, it's better. Fixed. Thanks.